packages/storage-macro/Cargo.toml

```diff
@@ -16,6 +16,5 @@ proc-macro = true
 syn = { version = "1.0.96", features = ["full"] }
 
 [dev-dependencies]
-cw-storage-plus = { version = "<=0.15.1, >=0.14.0", path = "../storage-plus" }
```

Comment:
Just FYI: the way the version requirements are specified here is a hack to make a cyclic dependency publishable to crates.io. Good thing we don't actually need it!

Reply:
Good that this is not needed, yes. Thanks for working on simplifying this.